src/test/java/com/ft/jerseyhttpwrapper/continuation/ExponentialBackoffContinuationSessionTest.java

```diff
@@ -98,7 +98,7 @@ public void shouldBackOffExponentially() {
     // interrupt the the thread almost immediately
     threadUnderTest.start();
 
-    sleepFor(5);
+    sleepFor(15);
```

Comment:
What's the reason behind changing this value?

Reply:
this one is ok during circleci build - there is no drama with it